conf/repo_1.toml

```diff
@@ -0,0 +1,40 @@
+[[repo]]
```

Comment:
When separate configs into single files by repo, I'm not sure whether keep the repo array here a good idea. Array with one item only can be redundant, but it's also useful when we want group some repos in one config file. Let's keep this buy now.